File: Programming/Assignment/Conditional Logic Progamms/Leap Year.c
#include<stdio.h>
main()
{
	int year;
	printf("Enter Year : ");
	scanf("%d",&year);
	
	if(year%4==0)
	{
		printf("Leap Year..");
	}
	else
	{
		printf("Common Year..");
	}
		
}

File: Programming/Assignment/Dart Assignment/Leap year.dart
import 'dart:io';

void main(List<String> args) {
  print('Enter year: ');
  var y = int.parse(stdin.readLineSync()!);

  if (y % 4 == 0) {
    print('$y is Leap year');
  } else {
    print('$y is not leap year');
  }
}
